src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -998,7 +998,10 @@ Made with 💗 by Microsoft and the PowerToys community.</value>
     <value>Filename parameters</value>
   </data>
   <data name="ColorModeHeader.Header" xml:space="preserve">
-    <value>App theme</value>
+    <value>Zone appearance</value>
+  </data>
+	<data name="ColorModeHeader.Description" xml:space="preserve">
+    <value>Customize the way zones look</value>
```

Comment:
@niels9001 
Isn't this the general hedar used on other pages too? We should introduce our own for fz Page.

Reply:
Yes, figured this out only after merging the PR.
Will fix this.